### ISSUE TITLE
[Mod Request] Better Advancements

### ISSUE THREAD

**Issue by osfanbuff63** [catppuccin/minecraft#30]
For some reason this mod overrides the default advancement textures (way2muchnoise/BetterAdvancements#139), so it needs special support, as outlined [here](https://github.com/way2muchnoise/BetterAdvancements/wiki/Creating-texture-sheets-for-Better-Advancements). It'd be great if support could be added, as it's a very cool mod to have.

**Comment by JustLetterV** [comment]
It should already be supported since version 1.1.0, although now that I took a look at the [Better Advancements Wiki](https://github.com/way2muchnoise/BetterAdvancements/wiki/Creating-texture-sheets-for-Better-Advancements) I noticed that I have missed the greyscale versions of textures that are used when changing color values in config. I'll add those in the future. Also, the advancement tab title has a partially wrong text color. Otherwise, the resource pack should work with better advancements. If it doesn't work let me know.
